yt_dlp/extractor/twitch.py

```diff
@@ -455,20 +455,28 @@ def _extract_info_gql(self, info, item_id):
             vod_id = 'v%s' % vod_id
         thumbnail = url_or_none(info.get('previewThumbnailURL'))
         is_live = None
+        thumbnails = None
         if thumbnail:
             if re.findall(r'/404_processing_[^.?#]+\.png', thumbnail):
-                is_live, thumbnail = True, None
+                is_live, thumbnails = True, None
             else:
                 is_live = False
-                for p in ('width', 'height'):
-                    thumbnail = thumbnail.replace('{%s}' % p, '0')
+                # for p in ('width', 'height'):
+                    # thumbnail = thumbnail.replace('{%s}' % p, '0')
+                common_res = [(0, 0), (160, 90), (320, 180), (480, 720), (640, 360), (768, 432), (1024, 576), (1280, 720), (1366, 768), (1920, 1080)]
```

Comment:
This is not what I had meant. I was asking only to return 0x0 along with whatever we extract. But this is fine too, assuming all these thumbs are available for all the videos. is it?

Reply:
According to [the reference](https://dev.twitch.tv/docs/api/reference/#get-streams), the height and width can be replaced by any value (max 2048\*2048 after the test), and 0\*0 is the quality of the video itself.
Even if the quality of vod is 1280\*720, it is possible to obtain a 1920*1080 size thumbnail.
Is it better to output 1920\*1080 thumbnails or 0\*0(=1280\*720) in this case?
If only want to return the picture quality of the vod itself, then modify it to
```py
thumbnails = [
    {'url': re.sub(r'\d+x\d+\.jpg', '0x0.jpg', thumbnail)},
    {'url': thumbnail}
]
```